sample-access-log.txt

```diff
@@ -0,0 +1,38 @@
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:07 +0000] 67.85.62.235 - 6DDE8EE6E7C8DB9C WEBSITE.GET.OBJECT kitties.js "GET /kitties.js HTTP/1.1" 200 - 75 75 225 225 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - XV1XIng8Qc2cVaeAJqln1CCWsDONnHWNhTO4mTr94r7VkF88V4pmSFWV0O0i78PCH0+lW4W0ETU= - - - cryptokitty-phrenology.buoydontfloat.com -
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:07 +0000] 67.85.62.235 - 939F721A71753258 WEBSITE.GET.OBJECT images/1399245.png "GET /images/1399245.png HTTP/1.1" 200 - 386765 386765 16 14 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - lDZlcT2kDdojsFxH1xGIjpBq7/oZzOxiGky1IZTZnZtVVK1g0NhYnMcTXiPTJ0jrtezvU+xuIjY= - - - cryptokitty-phrenology.buoydontfloat.com -
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:08 +0000] 67.85.62.235 - 06B27CA31C900584 WEBSITE.GET.OBJECT index.html "GET / HTTP/1.1" 200 - 2620 2620 10 9 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - kQ3Fsl9mSzvI3x/Ua23YW2L2XQrCfFdcqArywVwBIcnJGgPy/pEeyWlP4XEueg48KDehXQvaeLs= - - - cryptokitty-phrenology.buoydontfloat.com -
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:08 +0000] 67.85.62.235 - 491B6739EB0DB9BA WEBSITE.GET.OBJECT application.css "GET /application.css HTTP/1.1" 200 - 294 294 12 11 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - Mx15WWnMCMJJcAOtRwuEVbDbF6Kqa0CttoicqRl9DwYt1XUF0j5KvorEJKKgYeSb6iIOisFxRcI= - - - cryptokitty-phrenology.buoydontfloat.com -
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:08 +0000] 67.85.62.235 - 962BA0AB6AF1B732 WEBSITE.GET.OBJECT i.gif "GET /i.gif?kittySrc=1399245.png&kittyIndex=1&answer=yes HTTP/1.1" 200 - 42 42 7 7 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - m1UKy2riiiNB6Ww9sntDmIHv5dwLZ6SajOc/ZEdDtVvpZ5wemhP6qCglwFPdo1QyNm7MHZa5w+0= - - - cryptokitty-phrenology.buoydontfloat.com -
```

Comment:
Basically, we can parse these from all the logs:

```
"GET /i.gif?kittySrc=1399245.png&kittyIndex=1&answer=yes HTTP/1.1"
```

Reply:
aarrarhgghadghaidogh what is this

Reply:
Do you already have data collected? I'm pretty sure we can just use firebase client-side.